sql/moz-fx-data-shared-prod/domain_metadata_derived/top_domains_v1/query.sql

```diff
@@ -0,0 +1,78 @@
+WITH daily_tranco AS (
+  SELECT
+    rank,
+    NET.REG_DOMAIN(domain) AS domain,
+    NET.HOST(domain) AS host
+  FROM
+    `tranco.daily.daily`
+  WHERE
+    date IN (SELECT MAX(date) FROM `tranco.daily.daily`)
```

Comment:
Should this (and below, line 21) use `@submission_date`? IIUC these are not idempotent and may result in different values when re-run for the same day.

If this isn't supposed to be idempotent, consider dropping the `@submission_date` altogether (not sure it makes sense for something that's not being submitted to the pipeline).

Reply:
I've included submission_date in the output so we can partition the results by scheduled run

Reply:
Could do for Tranco: `WHERE date = @submission_date`, and for CrUX `WHERE PARSE_DATE('%Y%m', CAST(yyyymm AS STRING)) = @submission_date`, however I have no clue how this dataset is being used so take that with a grain of salt :)

If it's mostly "need to find the most recent value for any domain/host", should this just overwrite every time with the most recent daily/monthly data from tranco/crux?

Reply:
yeah, I was thinking that idempotency is less importance here than reliability. Given that these datasets are outside our control we can't guarantee what the partition dates will be at any given time. I would vote to leave them dynamically computed and know that we lose idempotency with that.